cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,85 @@
+= VPC Upgrade to Private Space 
+
+//Overview
+CloudHub VPC upgrade is an out-of-the-box tool and guided UI experience for an infrastructure upgrade to accelerate the migration of a VPC into a Private Space.
+
+// Benefits
+This tool enables you to provision the infrastructure of a CloudHb 2.0 Private Space by cloning the existing configurations of eligible CloubHub VPCs, and carrying over any existing network connections, including TGWs and VPNs. 
```

Comment:
The VPC to Private Space Upgrade tooling enables you to accelerate the creation of a CloudHub 2.0 Private Space by simply upgrading your existing CloudHub VPC. If you upgrade your CloudHub VPC into a Private Space instead of creating a new CloudHub 2.0 Private space, you can eliminate the time spent in acquiring new CIDR block, setting up new network connections, and configuring the private space from scratch. 

The tool enables you to provision the infrastructure of a CloudHub 2.0 Private Space by cloning the existing configurations of an eligible CloubHub VPC while leveraging the same CIDR block. If your VPC has existing TGW or VPN network connections, they will be carried over to the upgraded CloudHub 2.0 Private Space, and available for communication with any new applications deployed into the upgraded CloudHub 2.0 Private Space. 

Once your Private Space is created, proceed to incrementally migrate your applications from the CloudHub 1.0 VPC to the CloudHub 2.0 private space in a timely manner.